projects/packages/analyzer/scripts/example.php

```diff
@@ -81,7 +81,6 @@
 echo "\n*** Summary of dependencies by external file\n";
```

Comment:
Line 18 of this file has an instance of https://phabricator.wikimedia.org/T273574 too. Alas GitHub won't let me comment there, sigh.